src/pages/Startpage.tsx

```diff
@@ -16,25 +17,23 @@ const Startpage = ({
 }): JSX.Element => {
   return (
     <>
-      {
-        loggedin ? (
-          <div>
-            <Wrapper>
-              <>
-                <Desc>
-                  <div className="desc-title">Dollars & $ense Reality Fair</div>
-                  <DescText>
-                    The United Way of Northeast Louisiana Dollars & $ense
-                    Reality Fair is a financial education simulation during
-                    which high school students actively learn how to make better
-                    financial decisions and gain knowledge of budgeting, saving,
-                    and spending.
-                  </DescText>
-                </Desc>
-                {/* <Logoutbutton onClick={() => logout()}> sign out </Logoutbutton> */}
-              </>
-            </Wrapper>
-          </div>
+      {true ? (
```

Comment:
Did you mean to have `true` or is that debug?

Reply:
I thought I didnt commit this. My bad